source/iface/usm_allocations.rst

```diff
@@ -104,3 +297,25 @@ Output:
 
 .. literalinclude:: /examples/std-vector.out
    :lines: 5-
+
+.. _usm-implicit-data-movement:
+
+.. rubric:: Example 2
```

Comment:
Should we use ``using namespace sycl;`` in examples, or explicitly use ``sycl::``?

I believe I previously removed this line, but it shouldn't be a problem, we can select the preferred style and use it after Cycle 2 of rework.

Reply:
This line is also absent in Example 1.